input/vocabulary/valueset-condition-clinical-status.json

```diff
@@ -0,0 +1,95 @@
+{
+    "resourceType": "ValueSet",
+    "id": "condition-clinical",
+    "meta": {
+        "lastUpdated": "2019-11-01T09:29:23.356+11:00",
+        "profile": [
+            "http://hl7.org/fhir/StructureDefinition/shareablevalueset"
+        ],
+        "versionId": "bb0eb259-40b9-477b-9755-307e6a309b18"
+    },
+    "text": {
+        "status": "generated",
+        "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\"><h2>Condition Clinical Status Codes</h2><div><p>Preferred value set for Condition Clinical Status.</p>\n</div><p>This value set includes codes from the following code systems:</p><ul><li>Include all codes defined in <a href=\"codesystem-condition-clinical.html\"><code>http://terminology.hl7.org/CodeSystem/condition-clinical</code></a></li></ul></div>"
+    },
+    "extension": [
+        {
+            "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-wg",
+            "valueCode": "pc"
+        },
+        {
+            "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status",
+            "valueCode": "trial-use"
+        },
+        {
+            "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm",
+            "valueInteger": 3
+        }
+    ],
+    "url": "http://hl7.org/fhir/ValueSet/condition-clinical",
+    "identifier": [
+        {
+            "system": "urn:ietf:rfc:3986",
+            "value": "urn:oid:2.16.840.1.113883.4.642.3.164"
+        }
+    ],
+    "version": "4.0.1",
+    "name": "ConditionClinicalStatusCodes",
+    "title": "Condition Clinical Status Codes",
+    "status": "draft",
+    "experimental": false,
+    "date": "2019-11-01T09:29:23+11:00",
+    "publisher": "FHIR Project team",
+    "contact": [
+        {
+            "telecom": [
+                {
+                    "system": "url",
+                    "value": "http://hl7.org/fhir"
+                }
+            ]
+        }
+    ],
+    "description": "Preferred value set for Condition Clinical Status.",
+    "immutable": true,
+    "compose": {
+        "include": [
+            {
+                "system": "http://terminology.hl7.org/CodeSystem/condition-clinical"
+            }
+        ]
+    },
+    "expansion": {
+        "timestamp": "2019-11-01T09:29:23+11:00",
+        "contains": [
+            {
+                "code": "active",
```

Comment:
Need System for each code

Reply:
Updated the System to the valuset.Expansion.Contains

https://github.com/enjoysparkling/signal-mso-fhir-profiles/commit/d83e7b2ec27bf1ff25168f388d64a1ee78e576b0 Kindly review.